tutorials/pyMBE_tutorial.ipynb

```diff
@@ -1137,11 +2442,24 @@
+   "execution_count": 36,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "FileNotFoundError",
+     "evalue": "[Errno 2] No such file or directory: '/home/pablo/Escritorio/pyMBE/reference_parameters/interaction_parameters/Lunkad2021.txt'",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mFileNotFoundError\u001b[0m                         Traceback (most recent call last)",
+      "Cell \u001b[0;32mIn[36], line 1\u001b[0m\n\u001b[0;32m----> 1\u001b[0m \u001b[43mpmb\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mload_interaction_parameters\u001b[49m\u001b[43m \u001b[49m\u001b[43m(\u001b[49m\u001b[43mfilename\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43m \u001b[49m\u001b[43mpmb\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mget_resource\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43mreference_parameters/interaction_parameters/Lunkad2021.txt\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m)\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m      2\u001b[0m pmb\u001b[38;5;241m.\u001b[39madd_bonds_to_espresso (espresso_system \u001b[38;5;241m=\u001b[39m espresso_system)\n",
+      "File \u001b[0;32m~/Escritorio/pyMBE/pyMBE.py:1836\u001b[0m, in \u001b[0;36mpymbe_library.load_interaction_parameters\u001b[0;34m(self, filename, verbose)\u001b[0m\n\u001b[1;32m   1834\u001b[0m without_units \u001b[38;5;241m=\u001b[39m [\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mq\u001b[39m\u001b[38;5;124m'\u001b[39m,\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mes_type\u001b[39m\u001b[38;5;124m'\u001b[39m,\u001b[38;5;124m'\u001b[39m\u001b[38;5;124macidity\u001b[39m\u001b[38;5;124m'\u001b[39m]\n\u001b[1;32m   1835\u001b[0m with_units \u001b[38;5;241m=\u001b[39m [\u001b[38;5;124m'\u001b[39m\u001b[38;5;124msigma\u001b[39m\u001b[38;5;124m'\u001b[39m,\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mepsilon\u001b[39m\u001b[38;5;124m'\u001b[39m]\n\u001b[0;32m-> 1836\u001b[0m \u001b[38;5;28;01mwith\u001b[39;00m \u001b[38;5;28;43mopen\u001b[39;49m\u001b[43m(\u001b[49m\u001b[43mfilename\u001b[49m\u001b[43m)\u001b[49m \u001b[38;5;28;01mas\u001b[39;00m f:\n\u001b[1;32m   1837\u001b[0m     \u001b[38;5;28;01mfor\u001b[39;00m line \u001b[38;5;129;01min\u001b[39;00m f:\n\u001b[1;32m   1838\u001b[0m         \u001b[38;5;28;01mif\u001b[39;00m line[\u001b[38;5;241m0\u001b[39m] \u001b[38;5;241m==\u001b[39m \u001b[38;5;124m'\u001b[39m\u001b[38;5;124m#\u001b[39m\u001b[38;5;124m'\u001b[39m:\n",
+      "\u001b[0;31mFileNotFoundError\u001b[0m: [Errno 2] No such file or directory: '/home/pablo/Escritorio/pyMBE/reference_parameters/interaction_parameters/Lunkad2021.txt'"
+     ]
+    }
+   ],
+   "source": [
+    "pmb.load_interaction_parameters (filename = pmb.get_resource('parameters/peptides/Lunkad2021.txt'))\n",
```

Comment:
Consider clearing the notebook cell outputs. If we need notebooks with cell outputs, we can try adding an extra step in the CI/CD pipeline to execute the notebooks and push them as static HTML files to the GitHub Pages.

Reply:
@jngrad good catch! this was unintended, I will clean the cell outputs before merging.